packages/typespec-python/generator/pygen/codegen/serializers/model_serializer.py

```diff
@@ -17,7 +17,8 @@ def _documentation_string(prop: Property, description_keyword: str, docstring_ty
     sphinx_prefix = f":{description_keyword} {prop.client_name}:"
     description = prop.description(is_operation_file=False).replace("\\", "\\\\")
     retval.append(f"{sphinx_prefix} {description}" if description else sphinx_prefix)
-    retval.append(f":{docstring_type_keyword} {prop.client_name}: {prop.type.docstring_type()}")
+    docstring_type = prop.type.docstring_type(is_multipart_file_input=prop.is_multipart_file_input)
+    retval.append(f":{docstring_type_keyword} {prop.client_name}: {docstring_type}")
```

Comment:
we shouldn't need to pass `is_multipart_file_input` to the prop's type's docstrings. Instead, if we update the docstring in `model Property` to add the information if `self.is_multipart_file_input`, we don't need to pas this kwarg to `docstring_type`

Reply:
Updated. But I think it is not elegant: https://github.com/Azure/autorest.python/pull/2740/commits/89caa85f4511e191dfccec1057ecd66e66665716

Reply:
i agree to put all logic in `docstring_type` of the property type.